openmls/src/group/mls_group/builder.rs

```diff
@@ -136,7 +136,8 @@ impl MlsGroupBuilder {
             .map_err(LibraryError::unexpected_crypto_error)?;
 
         let message_secrets_store = MessageSecretsStore::new_with_secret(
-            self.max_past_epochs.unwrap_or_default(),
+            self.max_past_epochs
+                .unwrap_or(mls_group_create_config.max_past_epochs()),
```

Comment:
Sorry I neglected to look properly at this PR before creating #1662.

I'd propose that instead of just setting `self.max_past_epochs` here to the value in the config, we remove `self.max_past_epochs` entirely and just use the value in the config during group creation (as sketched in #1662). That would seem simpler to me. Or am I missing something?

Reply:
Yeah, let's use yours. I removed their fix from this PR.

Reply:
Oh, yeah I can also cherry-pick it, hang on